spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/bigquery/GcpBigQueryAutoConfiguration.java

```diff
@@ -57,6 +57,11 @@ public class GcpBigQueryAutoConfiguration {
 
   private int threadPoolSize;
 
+  private String universeDomain;
+  private String jsonWriterEndpoint;
+
+  private String host;
```

Comment:
Calling out the addition of this property since this is a deviation from our original design of exposing universe domain and endpoint.  Apiary libraries such as Bigquery and Storage only allow for customizing `host` (which is essentially endpoint without the port) instead of `endpoint` so we are doing the same in the Spring Cloud GCP layer. For context: https://github.com/googleapis/sdk-platform-java/pull/2329

@burkedavison @blakeli0